src/components/dls/Forms/Combobox/ComboboxItem.tsx

```diff
@@ -47,23 +47,24 @@ const ComboboxItem: React.FC<Props> = ({
           styles.itemContainer,
           { [styles.disabledItem]: disabled },
           { [styles.enabledItem]: !disabled },
+          { [styles.checkedItem]: checked },
         )}
       >
         <input
-          type="radio"
+          type="checkbox"
```

Comment:
should this be conditional radio/conditional based on the multiselect prop?

Reply:
This is done intentionally as before, there was an issue even with single select which is once you select you cannot un-select due it being a radio and not a checkbox.